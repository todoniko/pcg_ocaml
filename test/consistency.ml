let test_output ~f ~seed ~name =
  Pcg.seed seed;
  Printf.printf "# Few results of Pcg.%s with seed 0x%X\n" name seed;
  for i = 1 to 7 do
    Printf.printf "%i - 0x%016X\n" i (f ())
  done

let _ = test_output ~f:Pcg.xsh_rs ~seed:42 ~name:"xsh_rs"
