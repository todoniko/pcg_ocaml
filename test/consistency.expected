# Few results of Pcg.xsh_rs with seed 0x2A
1 - 0x00000000DEBFF77F
2 - 0x0000000054B00B9C
3 - 0x00000000DED17109
4 - 0x00000000383D10FA
5 - 0x00000000B7D5E650
6 - 0x00000000D8C19FA9
7 - 0x000000007300E1B7
